message_ix/model/MESSAGE/parameter_def.gms

```diff
@@ -8,63 +8,68 @@
 *
 * In |MESSAGEix|, all parameters are understood as yearly values, not as per (multi-year) period.
 * This provides flexibility when changing the resolution of the model horizon (i.e., the set ``year``).
-*
-* Parameters written in *italics* are auxiliary parameters
-* that are either generated automatically when exporting a ``message_ix.Scenario`` to gdx
-* or that are computed during the pre-processing stage in GAMS.
 ***
 
 ***
 * General parameters of the |MESSAGEix| implementation
 * ----------------------------------------------------
 *
+* .. caution::
+*    Parameters written in **bold** are auxiliary parameters
+*    that are either generated automatically when exporting a ``message_ix.Scenario`` to gdx
```

Comment:
I recently learned that it's sphinx auto-formats classes in **bold** rather than ``code`` - should we switch to that logic?

Reply:
Hi again @danielhuppmann,

I'm not sure if I understood what you meant with this sentence, but please check commit a8b7aca5a46e539a16367e887f88469c9058387c too see what I did to solve it. I think it is much better now since by clicking you are directly redirected to the other pages of the documentation.

Reply:
I please tell me if you think that further changes would be needed. Otherwise In my opinion is ready to merge 😃

Reply:
yes!